Snippets/Core/Core_5/Installers.cs

```diff
@@ -32,6 +33,9 @@ public static void Main(string[] args)
             if (runInstallers)
             {
                 busConfiguration.EnableInstallers();
+                // This will run the installers but not start the instance.
+                Bus.Create(busConfiguration);
+                return; // Exit application
```

Comment:
could we maybe just remove this and get rid of the resharper warning comment at the top of the file? or use Environment.Exit or so?